utils/selectors/createSafePage.js

```diff
@@ -13,8 +13,8 @@ export const createSafePage = {
   step_three: "p[data-testid='create-safe-step-three']",
   review_safe_name: "p[data-testid='create-safe-review-name']",
   review_req_conf: (ownerAmount = 1) => `p[data-testid='create-safe-review-req-owners-${ownerAmount}']`,
-  review_owner_name: (index = 0) => `p[data-testid='create-safe-owner-name-${index}']`,
-  review_owner_address: (index = 0) => `p[data-testid='create-safe-owner-address-${index}']`,
+  review_owner_name: (index = 0) => `div[data-testid='create-safe-owner-details-${index}'] div div p`,
+  review_owner_address: (index = 0) => `div[data-testid='create-safe-owner-details-${index}'] div div div p`,
```

Comment:
Can we skip the nested divs and just target the p?
Like this:
```
div[data-testid='create-safe-owner-details-${index}'] p
```

Reply:
We are using those divs as identifiers to select between name and address. If not we would have to add more data-testids.

Reply:
Oh, OK, both are in a <p> element. 🆗